plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -3975,8 +3999,20 @@ private void testTimestampCoercionOnCreateTable(@Language("SQL") String actualVa
         }
     }
 
+    private void testCharCoercionOnCreateTable(@Language("SQL") String actualValue, @Language("SQL") String expectedValue)
+    {
+        try (TestTable testTable = new TestTable(
+                getQueryRunner()::execute,
+                "test_char_coercion_on_create_table",
+                "(vch VARCHAR)")) {
```

Comment:
nit: `vch` isn't a general column name in this repository. I would rename to `col` or just `x`. Same for other places.